pkgs/tools/audio/goxlr-utility/default.nix

```
@@ -0,0 +1,65 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, installShellFiles
+, pkg-config
+, libpulseaudio
+, dbus
+, speechd
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "goxlr-utility";
+  version = "v0.12.6";
+
+  src = fetchFromGitHub {
+    owner = "GoXLR-on-Linux";
+    repo = pname;
+    rev = version;
+    sha256 = "vvaKCsqncRhag8IrS0AIfNqNHGU2WIvFaYISEVfUB2Y=";
```

hash is more modern:

```suggestion
    hash = "sha256-vvaKCsqncRhag8IrS0AIfNqNHGU2WIvFaYISEVfUB2Y=";
```